bin/ci-publish.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -ex
+
+npmPublish() {
+  npm publish
+}
+
+currentVersion() {
+  grep version <package.json | sed 's/[^0-9.]*\([0-9.]*\).*/\1/'
+}
+
+currentProject() {
+  npm ls --depth=-1 | head -1 | cut -f 1 -d " " | sed 's/@.*//'
+}
+
+checkVersion() {
+  local gitTag
+  gitTag=$(git describe --tags)
+
+  if [ "$(currentVersion)" != "$gitTag" ]; then
```

Comment:
I thought that the problem is that every PR tries to deploy. If HEAD is not tagged with a version tag that is the same as the version in package.json, we don't publish.